integration-cli/docker_utils.go

```diff
@@ -1046,11 +1046,12 @@ func getContainerState(c *check.C, id string) (int, bool, error) {
 	return exitStatus, running, nil
 }
 
-func buildImageCmd(name, dockerfile string, useCache bool) *exec.Cmd {
+func buildImageCmd(name, dockerfile string, useCache bool, buildArgs ...string) *exec.Cmd {
 	args := []string{"-D", "build", "-t", name}
 	if !useCache {
 		args = append(args, "--no-cache")
 	}
+	args = append(args, buildArgs...)
```

Comment:
oh man `buildArgs` is a misnomer here :). With this change I really wanted to allow these utility functions accept builder command line flags.

Sorry for the confusing name, I will rename it to `buildFlags`